tests/Aspire.Hosting.Tests/PublishAsDockerfileTests.cs

```diff
@@ -0,0 +1,73 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting.Tests;
+
+public class PublishAsDockerfileTests
```

Comment:
Change all of these assertions to match the new style, verifying the entire string.